### DISCUSSION TITLE
Has anyone implemented repeat types for this calendar?

### DISCUSSION THREAD

**Discussion by spacerobotTR** [excid3/simple_calendar#270] [General]
Has anyone ever done or are there plans to add repeat types to the events? Like if I wanted to schedule an event between start_date and end_date but only have it be MON-FRI or SAT-SUN only, or a recurring weekly or yearly event. Things like that. 

Any thoughts on how to do it?

**Comment by excid3** [comment]
Just supply the calendar with the generated events. Try montrose, ice_cube, etc or build your own.
